.github/ISSUE_TEMPLATE/LICENSE.md

```diff
@@ -0,0 +1,57 @@
+**Nightshade Vexillum License**
+
+```
+This license applies to the following directory:
+.github/ISSUE_TEMPLATE
+```
+
+```
+Emoji are used within this document to outwardly express conventions of easy-to-read documents, and/or help other cultures read the text at their convenience, if some words may be unclear to them.
+The Emoji contained within these documents are may not wholly represent the associated meaning, and the First Party does not warrant the misunderstanding of such.
+```
+
+## 📖 Definitions
+
+#### **SOFTWARE**
+
+The Software is the source-code, executable files, documentation and all associated documents, found within the directory `.github/ISSUE_TEMPLATE`
+
+#### **FIRST PARTY**
+
+The First Party are the Copyright Holders of the Software, as defined in **1️⃣ Section I: Parties**.
+
+#### **THIRD PARTY**
+
+The Third Party is any party that is **not** the First Party - see the **📖 Oxford English Dictionary** for details.
+
+#### **Eligible Party**
+
+An Eligible Party is a party that has been accepted by the First Party. By default manner, the First Party automatically accepts any party that wishes to use the software with no medium of communication other than this license required. The First Party may restrict this license for a Third Party, under any period of time, per their discretion.
+
+---
+
+### 1️⃣ Section I: Parties
+
+Copyright (c) Riverside Valley Corporation (the "First Party")
```

Comment:
Not copyrighted with Riverside Valley Corp.

Reply:
It is. The Issue Template was written by staff. JitHub is not copyright by Riverside, but the ISSUE_TEMPLATES are.